packages/cli/src/lifecycles/bundle.js

```diff
@@ -163,11 +163,13 @@ async function bundleStyleResources(compilation, resourcePlugins) {
 
 async function bundleApiRoutes(compilation) {
   // https://rollupjs.org/guide/en/#differences-to-the-javascript-api
-  const [rollupConfig] = await getRollupConfigForApis(compilation);
+  const apiConfigs = await getRollupConfigForApis(compilation);
 
-  if (rollupConfig.input.length !== 0) {
-    const bundle = await rollup(rollupConfig);
-    await bundle.write(rollupConfig.output);
+  if (apiConfigs.length > 0 && apiConfigs[0].input.length !== 0) {
+    apiConfigs.forEach(async rollupConfig => {
```

Comment:
If we want to do something `async` in a loop, we should use a `for ... of` or `for ... in` instead.   `forEech` is synchronous and combining these two can lead to unexpected behaviors (something I would like to more thoroughly clean up in #823 )
```js
for (const rollupConfig of apiConfigs) {
  const bundle = await rollup(rollupConfig);
  await bundle.write(rollupConfig.output);
}
```